src/Raygun.Blazor/Raygun.Blazor.csproj

```diff
@@ -44,6 +44,12 @@
         </PackageReference>
     </ItemGroup>
 
+    <ItemGroup Condition="'$(TargetFramework)' == 'net9.0'">
+        <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="[9.*, 10.0.0)" />
+        <PackageReference Include="Microsoft.Extensions.Http" Version="[9.*, 10.0.0)" />
+        <PackageReference Include="Microsoft.JSInterop" Version="[9.*, 10.0.0)" />
+    </ItemGroup>
+
     <ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">
         <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="[8.*, 9.0.0)" />
         <PackageReference Include="Microsoft.Extensions.Http" Version="[8.*, 9.0.0)" />
```

Comment:
Dependency constraints depend on the target framework. When targeting net 9.0, versions can be from `>= 9.0.0` to `<10.0.0`